MonoGame.Framework/Input/Touch/TouchLocation.cs

```diff
@@ -1,7 +1,7 @@
 #region License
 // /*
 // Microsoft Public License (Ms-PL)
-// MonoGame - Copyright � 2009-2010 The MonoGame Team
+// MonoGame - Copyright � 2009-2010 The MonoGame Team
```

Comment:
What happened here @Mindfulplays ?

Reply:
Codepage issues (apparently, VS Code says UTF-8!), just made it (C) :) note that GH diff shows/has codepage issues in the 'before' too for this file and the other ones, so fixed both.